lib/sorbet-rails/tasks/rails_rbi.rake

```diff
@@ -21,7 +21,7 @@ namespace :rails_rbi do
   desc "Generate rbi for rails models. Pass models name to regenrate rbi for only the given models."
   task models: :environment do |t, args|
     # need to eager load to see all models
-    Rails.application.eager_load!
+    Rails.configuration.eager_load_namespaces.each(&:eager_load!)
```

Comment:
Looking at the stackoverflow link & it looks like some people running into issue calling `eager_load!`. I think it might be safer to use `try`
```suggestion
    Rails.configuration.eager_load_namespaces.each { |s| s.try(:eager_load!) }
```